code/controllers/subsystem/SSjobs.dm

```diff
@@ -594,6 +594,8 @@ SUBSYSTEM_DEF(jobs)
 
 //fuck
 /datum/controller/subsystem/jobs/proc/CreateMoneyAccount(mob/living/H, rank, datum/job/job)
+	if(istype(job, /datum/job/cyborg) || istype(job, /datum/job/ai)) // They don't need bank accounts, robots don't get paid.
```

Comment:
Ideally, this could be a var on the job datum rather than hardcoded exceptions

Reply:
Done in upcoming commit